### PR TITLE
[Oav Pulgin] Update oav version to 0.4.1

### PR DESCRIPTION
**Pull request by vishrutshah**
Update oav plug-in to version 0.4.1

### CODE REVIEW

File: src/autorest-core/test/plugin.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ import { LoadLiterateSwagger } from "../lib/pipeline/swagger-loader";
       }
     });
     assert.strictEqual(await autoRest.Process().finish, true);
-    const expectedNumErrors = 7;
+    const expectedNumErrors = 2;
```
Comment (olydis) [inline src/autorest-core/test/plugin.ts]: do you know why this is? was oav reporting false positives?

Comment (vishrutshah) [inline src/autorest-core/test/plugin.ts]: Yup we fixed some bugs in oav like `integers` in query param, `encoding` checks and added support for polymorphic models that's why some of the errors disappeared. 

Ref: https://github.com/Azure/oav/issues?q=is%3Aissue+is%3Aclosed

Comment (olydis) [inline src/autorest-core/test/plugin.ts]: awesome!

**Comment by vishrutshah** [comment]
Thanks for the review!!

**Comment by olydis** [comment]
@vishrutshah *please* always squash and merge in the future!!!

**Comment by vishrutshah** [comment]
@olydis Of course. It's my bad. I thought there was some settings we can enable so that this does not happen even by mistake. I'll be careful but we should enable that settings :)

**Comment by olydis** [comment]
not sure about those settings, don't have access :) will ask Garrett when he's back

**Comment by vishrutshah** [comment]
@olydis I've enabled them. So now you should see message that says "Not enabled for this repository " in " Create a merge commit " option.

**Comment by olydis** [comment]
awesome, thank you! 🎉 🎈 🎆

**Pull request merged**